__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const db= require("../db/connection.js");
+const app= require("../app.js");
+const request= require("supertest");
+const seed= require("../db/seeds/seed")
+const testData= require("../db/data/test-data/index.js");
+const topics = require("../db/data/test-data/topics.js");
```

Comment:
Dont need this topics bit - all the test data should be seeded through the testData import above it